_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,489 @@
+
+```html
+<style>
+	p {
+		color: white;
+		background-color: hsl(232, 50%, 25%);
+	}
+	a {
+		color: white;
+		background-color: hsl(0, 50%, 25%);
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 4
+
+This element with a [semantic role][] that inherits from link, with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
+
+```html
+<style>
+	p {
+		color: hsl(0, 50%, 25%);
+		background-color: white;
+	}
+	a {
+		color: hsl(232, 50%, 25%);
+		background-color: white;
+		text-decoration: underline;
+	}
+</style>
+<p>This is indicated in the cost of a <a href="#bcc0f155" role="doc-glossref">credit default swap</a>.</p>
+<h1>Glossary</h1>
+<p id="bcc0f155">
+	A credit default swap (CDS) is a financial derivative or contract that allows an investor to "swap" or offset his or
+	her credit risk with that of another investor.
+</p>
+```
+
+#### Passed Example 5
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable color because the color contrast is 5.0:1.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: crimson;
+		background-color: white;
+		text-decoration: none;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a class="test" href="http://w3.org/WAI">WAI webpage</a>.
+</p>
+```
+
+#### Passed Example 6
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable bottom border when it is hovered or focused.
+
+```html
+<style>
+	p {
+		color: hsl(0, 50%, 25%);
+		background-color: white;
+	}
+	a {
+		color: hsl(232, 50%, 25%);
+		background-color: white;
+		text-decoration: none;
+	}
+	a:hover,
+	a:focus {
+		border-style: solid;
+		border-color: red;
+		border-width: 0px;
+		border-bottom-width: 1px;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 7
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable box-shadow.
+
+```html
+<style>
+	p {
+		color: hsl(0, 50%, 25%);
+		background-color: white;
+	}
+	a.test {
+		color: hsl(232, 50%, 25%);
+		background-color: white;
+		text-decoration: none;
+		box-shadow: 4px 4px;
+	}
+</style>
+<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 8
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable font-style when it is focused.
+
+```html
+<style>
+	p {
+		color: hsl(0, 50%, 25%);
+		background-color: white;
+	}
+	a {
+		color: hsl(232, 50%, 25%);
+		background-color: white;
+		text-decoration: none;
+	}
```

Comment:
Technique G183 has established a minimum contrast ratio of 3:1 to pass, when combined with a distinguishable hover style and a distinguishable focus style. However, the two colors in this example, hsl(0, 50%, 25%) and hsl(232, 50%, 25%), have a contrast ratio (relative luminance) of only 1.1:1.

This issue could be fixed one of these ways...

(1) Find a combination of two saturated foreground colors and one background color, such that the two foreground colors differ by at least 3:1 contrast ratio while also satisfying 1.4.3 Contrast (Minimum) against the background color. To conform to WCAG Level AA, I'm not sure if such a combination exists. It might only exist with large-scale text to meet 1.4.3. Or...

(2) Choose two saturated foreground colors which differ by at least 3:1 contrast ratio, but cannot meet 1.4.3 for any possible single background color and font size. Include a note stating that while this technique meets 1.4.1 and potentially WCAG Level A conformance, it cannot be used to achieve WCAG Level AA conformance because it will always fail 1.4.3. Or...

(3) Rewrite the overall rule so it addresses all color differences, not just hue differences, then choose two colors for this example with at least a 3:1 contrast ratio; or

(4) Let the overall rule continue to address hue differences only, and delete this passed example.

Reply:
I'm not sure what is the issue here. The colors were picked to have different hue while keeping to under 3:1 contrast ratio on purpose. That is because the example is showing one of the ways (a different font-style in this instance) that can be used to make the link different from the surrounding text besides the difference in color (hue).

Is this is fact the same issue that I replied to in https://github.com/act-rules/act-rules.github.io/pull/1010/files#r518651304 ?